lib/merit/controller_extensions.rb

```diff
@@ -46,16 +46,20 @@ def had_errors?
       target_object.respond_to?(:errors) && target_object.errors.try(:present?)
     end
 
-    def target_object
-      target_obj = instance_variable_get(:"@#{controller_name.singularize}")
-      if target_obj.nil?
+    def warn_no_object_found
         str = '[merit] No object found, you might need a ' \
           "'@#{controller_name.singularize}' variable in " \
           "'#{controller_path}_controller' if no reputation is applied. " \
           'If you are using `model_name` option in the rule this is ok.'
         Rails.logger.warn str
```

Comment:
This block has one extra level of indentation in the method. It should be moved back two characters.